packages/turf-bezier/index.js

```diff
@@ -13,6 +13,7 @@ var Spline = require('./spline.js');
  * @param {number} [resolution=10000] time in milliseconds between points
  * @param {number} [sharpness=0.85] a measure of how curvy the path should be between splines
  * @returns {Feature<LineString>} curved line
+ * @addToMap line, curved
```

Comment:
Maybe only showing the final curved line would be useful here. Having both would most likely clutter the example map and be confusing (it's hard to display a before & after with a single map).